Added additional requirement againts the audience of the Status Assertions and their Errors (#63)
peppelinux/draft-demarco-oauth-status-attestations: draft-demarco-oauth-status-attestations.md

set to "status-assertion+{jwt,cwt}" or "status-assertion-error+{jwt,cwt}", depending by the object type.
- The corresponding entry in the response MUST be of the same data format as requested. For example,
if the entry in the request is "jwt", then the entry at the same position in the response MUST also be "jwt".

> - The corresponding entry in the response MUST NOT contain the intended audience or any information regarding the Verifier to whom the assertion may be presented.

Author: ```suggestion
- The corresponding entry in the response MUST NOT contain any
information regarding the Verifier to whom it may be presented,
such as the Verifier identifier as the intended audience.
```

code set to 200 and the `status_assertion_responses` array with the related
Status Assertion Error object.

The Status Assertion Error MUST NOT be presented or provided to a Verifier,
the only audience of the Status Assertion Error is the Holder of the Credential
that has requested the Status Assertion. Therefore,

> the Status Assertion Error MUST NOT contain the parameter `aud`.

Reviewer: @peppelinux please help me to understand this logical implication (Therefore,...). I agree that the response MUST NOT to be presented to a Verifier, but saying that it MUST NOT contain the aud parameter seems to me strong. The logical implication is that
- the aud MAY be used
- if it is used MUST be only the client 
- the client MUST check that the aud parameter only contains the client.

In this way we force the response to be not presented to third party and the aud parameter could also be used as security check. WDYT?

Author: we were moved from a json error message to a jwt. Therefore I didn't see the value of adding so much requirements for the data schema of the errors.

I tend to reduce the options (may) aiming to give clear paths to the implementers. However, this requirement to keep out aud from the dataset is not critical to me. I would therefore remove the MUST NOT and not include at the same time the `aud` parameter definition in the assertion error data set

Author: @fmarino-ipzs do you agree? Do I have to remove that MUST NOT to get this PR merged?

Reviewer: I get your point. I think that forcing third parties to not consume the response jwt can be properly done by using aud parameter as it is stated in the [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3). But in this PR you are explicitly saying that aud claim MUST NOT to be used and this seems a little bit strange to me..
Anyway it is not critical for me as well so I'm going to approve this PR and we can came to this point later if you think it is valuable

> that has requested the Status Assertion. Therefore,
> the Status Assertion Error MUST NOT contain the parameter `aud`.

Author: ```suggestion
that has requested the Status Assertion. Therefore,
it is not necessary that the Status Assertion Error
contains the parameter `aud` within its payload.
```
